### ISSUE TITLE
add new dependencies

### ISSUE THREAD

**Issue by B1Sandmann** [sapcc/fedIPA#17]
IPA can use the `stress-ng` and `fio` package beginning with version 8.0 / 8.1 (currently unreleased) https://docs.openstack.org/releasenotes/ironic-python-agent/unreleased.html#relnotes-8-1-0 we should add them as well.

**Comment by killermoehre** [comment]
@B1Sandmann PRs are welcome.
Both [stress-ng](https://pkgs.org/download/stress-ng) and [fio](https://pkgs.org/download/fio) are included in Fedora.

Will this somehow change the deployment time? Or is this stress test of both disks and CPU optional?

**Comment by B1Sandmann** [comment]
yeah I just wanted to add it to the backlog as the version where this is needed isn't released yet. So I think adjusting the IPA version and updating the dependencies can be done in one PR. 

As for the deployment time: I assume it shouldn't change the deployment time (from my limited understanding) as this is only used in a special mode and not on every deployment

**Comment by killermoehre** [comment]
I created a project to track everything needed for IPA 8.*
